packages/venia-ui/lib/components/Mask/mask.js

```diff
@@ -13,49 +22,3 @@ const Mask = props => {
 };
 
 export default Mask;
-
-// /**
-//  * A component that masks content.
-//  *
-//  * @class Mask
-//  * @extends {Component}
-//  *
-//  * @typedef Mask
-//  * @kind class component
-//  *
-//  * @returns {React.Element} A React component that will mask content.
-//  */
-// class Mask extends Component {
-//     /**
-//      * Props for {@link Mask}
-//      *
-//      * @typedef props
-//      *
-//      * @property {Object} classes An object containing the class names for the
-//      * Mask component.
-//      * @property {string} classes.root classes for root container
-//      * @property {string} classes.root_action classes for root container if
-//      * the `isActive` property is `true`.
-//      * @property {Function} dismiss the handler for on the `onClick` event
-//      * handler.
-//      * @property {boolean} isActive whether the mask is in an active state
-//      * or not.
```

Comment:
These need to be added to the JSDocs as `@param` entries under the `props` parameter.

Example:

```
@param {callback} props.dismiss Handler for the onClick() event
@param {callback} props.isActive True if the mask is in an active state. False otherwise.
```